src/assets/styles/20-tools/_mixins/_grid.scss

```diff
@@ -172,118 +172,6 @@
   justify-self: $value;
 }
 
-/**
```

Comment:
Not used anywhere and for some reason this now causes sass errors when starting up Storybook